module-2/middlewares/check-jwt.ts

```diff
@@ -0,0 +1,21 @@
+import { Request, Response, NextFunction } from 'express';
+import AuthService from '../services/AuthService';
+
+export default function (req: Request, res: Response, next: NextFunction): void {
+  const auth = req.headers.authorization;
+
+  if (!auth) {
+    res.status(401).send({ message: 'Not authorized' });
+  } else {
+    const accessToken = auth
+      ?.split(' ')
+      .map(v => v.trim())
+      .filter(v => !/(bearer)/i.exec(v))[0];
+
+    if (AuthService.isAccessTokenValid(accessToken)) {
```

Comment:
Consider making this part and all of the similar parts in auth router controllers similar to each other.
E.g. POST /login looks confusingly different from the rest of the controllers.

Reply:
Done, thanks!